### ISSUE TITLE
How to set different learning rate to different layers with pretained model which only have symbols json file and params file?

### ISSUE THREAD

**Issue by shuokay** [apache/incubator-mxnet#8623]
Model downloaded from model zoo only have symbol json file and model params file. I know one way of setting different learning rate in python file for example:
```python
c1 = mx.sym.Convolution(data, kernel=(3,3), num_filter=16, name="c1", attr={"lr_mult": "0.01"})
```
However, the pretrained model dont have symbol  python file, so how can I set `lr_muli` attribute to it.
I think `net.get_internals()` may help, but it doesn't get the `c1` symbol but `c1_weights` and `c1_bias`

**Comment by shuokay** [comment]
After digging into source code, I think it is just `c1_weight._set_attr(lr_mult="0.01")` and `c1_bias._set_attr(lr_mult="0.01")`, is there anyone can confirm this?

**Comment by eric-haibin-lin** [comment]
Yes setting `lr_mult` attr for the variable will solve this issue.

**Comment by ShownX** [comment]
how to set different learning rate in Gluon?

**Comment by Light--** [comment]
is there any example for setting different learning rate for different layers in mxnet?

**Comment by szha** [comment]
> Yes setting lr_mult attr for the variable will solve this issue.

@Light-- this still applies to Gluon where there's a per-parameter attribute of `lr_mult`.

**Comment by Light--** [comment]
> @Light-- this still applies to Gluon where there's a per-parameter attribute of `lr_mult`.
then how to output the lr of one layer?

**Comment by szha** [comment]
the lr of each layer would be the `lr * lr_mult`

**Comment by Light--** [comment]
> the lr of each layer would be the `lr * lr_mult`

you mean i can't output it?

**Comment by szha** [comment]
right, I don't think it's exposed in the frontend. lr_mult should be set on a variable such as the weight of fullyconnected.

**Comment by Light--** [comment]
> right, I don't think it's exposed in the frontend. lr_mult should be set on a variable such as the weight of fullyconnected.

ok, since i can't find any official example for doing this, now i do it by myself:
1, is the following code right?
```
_weight = mx.symbol.Variable("fc1_weight", lr_mult=10, )
fc1 = mx.sym.FullyConnected(data=fc0, num_hidden=num_classes, name='fc1')
```
2, do i need to set the lr_mut for optimizer at the same time to make it work? or just set one of them is enough? [some post said it also need to set lr_mult for the optimizer](https://discuss.gluon.ai/t/topic/6002/12)

**Comment by szha** [comment]
> 2, do i need to set the lr_mult for optimizer at the same time to make it work?

I think it's only required in one of the two places. the approach in post2 is to use a dictionary to set the lr_mult for each variable by name according to the dictionary input.
